app/src/org/commcare/utils/LayoutDirectionUtilCompat.java

```diff
@@ -0,0 +1,34 @@
+package org.commcare.utils;
+
+import android.os.Build;
+import android.view.View;
+
+public class LayoutDirectionUtilCompat {
```

Comment:
Dont this this class is needed anymore since we moved the mirroring logic to xml